cloudify_vsphere/utils/__init__.py

```diff
@@ -45,7 +45,7 @@ def op(func):
     convention started by the `decorator` library).
     """
 
-    @operation
+    @operation(resumable=True)
```

Comment:
Is there anything that's unsafe to resume on?

Off the top of my head, VMs will be fine because they use the name and (usually) the instance suffix.
What about the other entities we support? There aren't any that create new entities that don't have names that will cause them to not be able to be idempotent?

Reply:
I have checked ippool, cdrom and network.